### PR TITLE
Spec coverage on Settings/ controllers specs

### PR DESCRIPTION
**Pull request by mjankowski**
Adds more coverage to the /settings/ controllers.

Does not quite get to 100% on this group ... but pretty close. Mostly failure edge cases left to do, but I think the full happy path is covered after these changes.

### CODE REVIEW

File: spec/controllers/settings/aliases_controller_spec.rb [hunk 1]
```diff
@@ -18,4 +18,45 @@
...
+    let(:account_alias) do
+      AccountAlias.new(account: user.account, acct: 'new@example.com').tap do |account_alias|
+        account_alias.save(validate: false)
+      end
+    end
+
+    it 'removes an alias' do
+      delete :destroy, params: { id: account_alias.id }
+
+      expect(response).to redirect_to(settings_aliases_path)
```
Comment (ClearlyClaire) [inline spec/controllers/settings/aliases_controller_spec.rb:52]: I think it should also check that the record actually gets deleted.

File: spec/controllers/settings/aliases_controller_spec.rb [hunk 2]
```diff
@@ -18,4 +18,45 @@
...
   end
+
+  describe 'POST #create' do
+    context 'with valid alias' do
+      before { stub_resolver }
+
+      it 'creates an alias for the user' do
+        post :create, params: { account_alias: { acct: 'new@example.com' } }
+
+        expect(response).to redirect_to(settings_aliases_path)
```
Comment (ClearlyClaire) [inline spec/controllers/settings/aliases_controller_spec.rb:29]: I think it should also check that a record gets actually created.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]
In most cases, the tests you added only check for the HTTP response code. While that's still an improvement given these methods aren't currently tested at all, that makes the test descriptions misleading, whereas similar test descriptions read something like “returns http success” in our other tests.

I'd like that either the tests get changed to actually test what they claim, or their description be changed to reflect what they are actually testing.

**Comment by mjankowski** [comment]
Pushed update with a few spec changes for better creation/deletion checks, and also some wording changes to make some of the descriptions less inaccurate.

You are correct that my current goal here is broad coverage and just getting all the lines executing in the spec. I'll continue to go deeper on actual behavior specs once the broad coverage is solid.

File: spec/controllers/settings/migration/redirects_controller_spec.rb [hunk 3]
```diff
@@ -17,4 +17,47 @@
...
+
+    context 'with non valid params' do
+      it 'does not update the account and renders the new page' do
+        post :create, params: { form_redirect: { acct: '' } }
+
+        expect(response).to have_http_status(200)
+        expect(response).to render_template(:new)
+      end
+    end
+  end
```
Comment (ClearlyClaire) [inline spec/controllers/settings/migration/redirects_controller_spec.rb:40]: I think those two tests still have misleading descriptions.

**Review by ClearlyClaire — CHANGES_REQUESTED** [review]

File: spec/controllers/settings/migration/redirects_controller_spec.rb (hunk 3 above)
Comment (mjankowski) [reply spec/controllers/settings/migration/redirects_controller_spec.rb]: They sure do -- pushed another commit changing their descriptions as well.

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**